packages/@aws-cdk/aws-stepfunctions-tasks/test/eks/call.test.ts

```diff
@@ -0,0 +1,103 @@
+import * as eks from '@aws-cdk/aws-eks';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { EksCall, HttpMethods } from '../../lib/eks/call';
+
+let stack: Stack;
+let cluster: eks.Cluster;
+
+beforeEach(() => {
+  //GIVEN
+  stack = new Stack();
+  cluster = new eks.Cluster(stack, 'Cluster', {
+    version: eks.KubernetesVersion.V1_18,
+    clusterName: 'eksCluster',
+  });
+});
+
+test('Call an EKS endpoint', () => {
+  // WHEN
+  const task = new EksCall(stack, 'Call', {
+    cluster: cluster,
+    httpMethod: HttpMethods.GET,
+    httpPath: 'path',
+    requestBody: sfn.TaskInput.fromObject({
+      RequestBody: 'requestBody',
+    }),
```

Comment:
should the type of `requestBody` be `sfn.TaskInput` ? it's currently not configured in that way.

Reply:
Yes, that's a good point. Changed the ```requestBody``` to be ```sfn.TaskInput```